src/components/SkillFeedbackPage/SkillFeedbackPage.js

```diff
@@ -0,0 +1,612 @@
+                        id="post-feedback"
+                        hintText="Skill Feedback"
+                        defaultValue=""
+                        errorText={this.state.errorText}
+                        multiLine={true}
+                        fullWidth={true}
+                      />
+                      <RaisedButton
+                        label="Post"
+                        labelStyle={{ color: 'white' }}
+                        backgroundColor={'#4285f4'}
+                        style={{ margin: 10 }}
+                        onClick={this.postFeedback}
+                      />
+                    </div>
+                  </div>
+                </div>
+              ) : null}
+              {feedbackCards &&
+                (feedbackCards.length > 0 ? (
+                  <List style={{ padding: '8px 0px 0px 0px' }}>
+                    {userFeedbackCard}
+                    {feedbackCards}
+                  </List>
+                ) : (
+                  <div className="feedback-default-message">
+                    No feedback present for this skill!
+                  </div>
+                ))}
+              <Dialog
+                title="Edit Feedback"
+                actions={editActions}
+                modal={false}
+                open={this.state.openEditDialog}
+                onRequestClose={this.handleEditClose}
+              >
+                <TextField
+                  id="edit-feedback"
+                  hintText="Skill Feedback"
+                  defaultValue={userFeedback}
+                  errorText={this.state.errorText}
+                  multiLine={true}
+                  fullWidth={true}
+                />
+              </Dialog>
+              <Dialog
+                title="Delete Feedback"
+                actions={deleteActions}
+                modal={true}
+                open={this.state.openDeleteDialog}
+                onRequestClose={this.handleEditClose}
+              >
+                Are you sure, you want to delete your feedback ?
+              </Dialog>
+            </Paper>
+          </div>
+          <Footer />
+        </div>
+      );
+    } else {
+      renderElement = (
+        <div>
+          <StaticAppBar {...this.props} />
+          <h1 className="skill_loading_container">
+            <div className="center">
+              <CircularProgress size={62} color="#4285f5" />
+              <h4>Loading</h4>
+            </div>
+          </h1>
+        </div>
+      );
+    }
+
+    return (
+      <div>
+        {renderElement}
+        <div>
+          <AuthorSkills
+            ref={c => {
+              this.author = c;
+            }}
+            open={this.state.showAuthorSkills}
+            close={this.closeAuthorSkills}
+            author={this.state.author}
+            authorUrl={this.state.author_url}
+          />
+        </div>
+      </div>
+    );
+  }
+}
+
+SkillFeedbackPage.propTypes = {
+  location: PropTypes.object,
```

Comment:
Not required.

Reply:
I think this is required, without this it was giving: 'location' is missing in props validation.